components/stimulation/StimulationStudioRepeatModal.vue

```diff
@@ -0,0 +1,237 @@
+  padding: 0px;
+  margin: 0px;
+  background: rgb(17, 17, 17);
+  position: absolute;
+  width: 522px;
+  height: 590px;
+  top: calc(55px - 55px);
+  left: calc(852px - 852px);
+  visibility: visible;
+  border: 2px solid rgb(0, 0, 0);
+  border-radius: 0px;
+  box-shadow: none;
+  pointer-events: all;
+  z-index: 5;
+}
+
+.span__stimulationstudio-current-settings-title {
+  pointer-events: all;
+  line-height: 100%;
+  transform: rotate(0deg);
+  overflow: hidden;
+  position: absolute;
+  width: 500px;
+  height: 30px;
+  top: calc(72px - 55px);
+  left: calc(863px - 852px);
+  padding: 5px;
+  visibility: visible;
+  user-select: none;
+  font-family: Muli;
+  font-weight: normal;
+  font-style: normal;
+  text-decoration: none;
+  font-size: 19px;
+  color: rgb(255, 255, 255);
+  text-align: center;
+}
+
+.canvas__stimulationstudio-horizontal-line-seperator-one {
+  transform: rotate(0deg);
+  pointer-events: all;
+  position: absolute;
+  width: 472px;
+  height: 2px;
+  top: calc(104px - 45px);
+  left: calc(878px - 852px);
+  visibility: visible;
+  background-color: #292929;
+  opacity: 0.5;
+}
+
+.span__stimulationstudio-current-settings-sub-title-one {
+  pointer-events: all;
+  line-height: 100%;
+  transform: rotate(0deg);
+  overflow: hidden;
+  position: absolute;
+  width: 499px;
+  height: 30px;
+  top: calc(122.5px - 55px);
+  left: calc(863px - 852px);
+  padding: 5px;
+  visibility: visible;
+  user-select: none;
+  font-family: Muli;
+  font-weight: normal;
+  font-style: normal;
+  text-decoration: none;
+  font-size: 19px;
+  color: rgb(255, 255, 255);
+  text-align: center;
+}
+
+.button-container {
+  top: 210px;
+  left: 0;
+  position: absolute;
+  cursor: pointer;
+}
+
+.button-container-mono {
+  top: 540px;
+  position: absolute;
+  cursor: pointer;
+}
+
+.span__stimulationstudio-current-settings-label-one {
+  pointer-events: all;
+  line-height: 100%;
+  transform: rotate(0deg);
+  overflow: hidden;
+  position: absolute;
+  width: 206px;
+  height: 30px;
+  top: calc(164.5px - 55px);
+  left: 45px;
+  padding: 5px;
+  visibility: visible;
+  user-select: none;
+  font-family: Muli;
+  font-weight: normal;
+  font-style: normal;
+  text-decoration: none;
+  font-size: 17px;
+  color: rgb(183, 183, 183);
+  text-align: right;
+}
+.div__stimulationstudio-duration-input-container {
+  pointer-events: all;
+  transform: rotate(0deg);
+  overflow: hidden;
+  position: absolute;
+  width: 162px;
+  height: 57px;
+  top: calc(151px - 55px);
+  left: 250px;
+  visibility: visible;
+}
+.span__stimulationstudio-duration-input {
+  overflow: hidden;
+  white-space: nowrap;
+  text-align: left;
+  font-weight: normal;
+  transform: translateZ(0px);
+  position: absolute;
+  width: 150px;
+  height: 50px;
+  line-height: 45px;
+  top: 6px;
+  left: 6px;
+  user-select: none;
+  font-family: Anonymous Pro;
+  font-style: normal;
+  text-decoration: none;
+  font-size: 17px;
+  color: rgb(255, 255, 255);
+}
+</style>
```

Comment:
mostly just reused code from the waveform settings modal, will enhance once we get more information on what is needed in it, currently just saves how number of repeats